models/articles.model.js

```diff
@@ -60,23 +41,52 @@ exports.pullArticlesWithCommentCount = () => {
     });
 };
 
-exports.pullArticleComments = (article_id) => {
+exports.updateArticleId = (article_id, body) => {
   return db
     .query(
-      `
-    SELECT article_id FROM articles WHERE article_id=$1
-    `,
-      [article_id]
+      `UPDATE articles
+        SET
+        votes= votes + $2
+        WHERE article_id = $1
+        RETURNING *;`,
+      [article_id, body.inc_votes]
     )
-    .then((results) => {
-      if (!results.rows.length) {
-        return Promise.reject({
-          status: 404,
-          msg: "404 - Article Not Found",
-        });
+    .then((result) => {
+      if (!result.rows.length) {
+        return Promise.reject({ status: 404, msg: "404 - Article Not Found" });
       } else {
-        console.table(results.rows);
-        return results.rows;
+        return result.rows[0];
       }
     });
 };
+
+exports.pullArticleComments = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then((result) => {
+      if (!result.rows.length) {
+        return Promise.reject({ status: 404, msg: "404 - Article Not Found" });
+      } else {
+        return result.rows[0];
+      }
+    })
+    .then((result) => {
+      return db
+        .query(
+          `SELECT comment_id, votes, created_at, author, body
+            FROM comments 
+            WHERE article_id = $1;`,
+          [article_id]
+        )
+        .then((result) => {
```

Comment:
This needs to be two separate functions. This function should get all the comments relating to an article, but not the article itself. That's for getARticleById to do!